### DISCUSSION TITLE
List of presets

### DISCUSSION THREAD

**Discussion by Sparklytical** [zombieFox/nightTab#275] [Show and tell]
# List of some presets from Reddit and GitHub

### Share your own presets on [/r/nightTab](https://www.reddit.com/r/nighttab/)

**How to [import these presets.](https://github.com/zombieFox/nightTab/wiki/Data-backup-and-restore#restore-data)**

| Preset 003 |
|-|
| [![Preset 003](https://raw.githubusercontent.com/zombieFox/nightTab/master/asset/screenshot/screenshot-003.png)](https://github.com/zombieFox/nightTab/blob/master/asset/screenshot/screenshot-003.json)| |
| [Download preset](https://github.com/zombieFox/nightTab/blob/master/asset/screenshot/screenshot-003.json) |

| Preset 004 |
|-|
| [![Preset 004](https://raw.githubusercontent.com/zombieFox/nightTab/master/asset/screenshot/screenshot-004.png)](https://github.com/zombieFox/nightTab/blob/master/asset/screenshot/screenshot-004.json)| |
| [Download preset](https://github.com/zombieFox/nightTab/blob/master/asset/screenshot/screenshot-004.json) |

| Preset 005 |
|-|
| [![Preset 005](https://raw.githubusercontent.com/zombieFox/nightTab/master/asset/screenshot/screenshot-005.png)](https://github.com/zombieFox/nightTab/blob/master/asset/screenshot/screenshot-005.json)| |
| [Download preset](https://github.com/zombieFox/nightTab/blob/master/asset/screenshot/screenshot-005.json) |

| Preset 006 |
|-|
| [![Preset 006](https://raw.githubusercontent.com/zombieFox/nightTab/master/asset/screenshot/screenshot-006.png)](https://github.com/zombieFox/nightTab/blob/master/asset/screenshot/screenshot-006.json)| |
| [Download preset](https://github.com/zombieFox/nightTab/blob/master/asset/screenshot/screenshot-006.json) |

| Preset 007 |
|-|
| [![Preset 007](https://raw.githubusercontent.com/zombieFox/nightTab/master/asset/screenshot/screenshot-007.png)](https://github.com/zombieFox/nightTab/blob/master/asset/screenshot/screenshot-007.json)| |
| [Download preset](https://github.com/zombieFox/nightTab/blob/master/asset/screenshot/screenshot-007.json) |

| Preset 008 |
|-|
| [![Preset 008](https://raw.githubusercontent.com/zombieFox/nightTab/master/asset/screenshot/screenshot-008.png)](https://github.com/zombieFox/nightTab/blob/master/asset/screenshot/screenshot-008.json)| |
| [Download preset](https://github.com/zombieFox/nightTab/blob/master/asset/screenshot/screenshot-008.json) |

| Preset 009 |
|-|
| [![Preset 009](https://raw.githubusercontent.com/zombieFox/nightTab/master/asset/screenshot/screenshot-009.gif)](https://github.com/zombieFox/nightTab/blob/master/asset/screenshot/screenshot-009.json)| |
| [Download preset](https://github.com/zombieFox/nightTab/blob/master/asset/screenshot/screenshot-009.json) |

| Preset 010 |
|-|
| [![Preset 010](https://raw.githubusercontent.com/zombieFox/nightTab/master/asset/screenshot/screenshot-010.png)](https://github.com/zombieFox/nightTab/blob/master/asset/screenshot/screenshot-010.json)| |
| [Download preset](https://github.com/zombieFox/nightTab/blob/master/asset/screenshot/screenshot-010.json) |

| Preset 011 |
|-|
| [![Preset 011](https://raw.githubusercontent.com/zombieFox/nightTab/master/asset/screenshot/screenshot-011.png)](https://github.com/zombieFox/nightTab/blob/master/asset/screenshot/screenshot-011.json)| |
| [Download preset](https://github.com/zombieFox/nightTab/blob/master/asset/screenshot/screenshot-011.json) |

| Minimal Coffee |
|-|
| [![Minimal Coffee](https://i.redd.it/ymjac7y4pyk51.png)](https://cdn.discordapp.com/attachments/704792091955429426/751129823610667148/zomfox_clean.json)| |
| [Download preset](https://cdn.discordapp.com/attachments/704792091955429426/751129823610667148/zomfox_clean.json) |
| [Reddit post](https://www.reddit.com/r/startpages/comments/ilx53r/minimal_coffee_nighttab/) |

| Persona 5 |
|-|
| [![Persona 5](https://preview.redd.it/hlpg8vel17h71.png?width=1920&format=png&auto=webp&s=122573f705ef80c5a3771ddd4f7c157bef52102a)](https://cdn.discordapp.com/attachments/401099306167435269/875851587971592222/Persona_5_Nighttab_final.json)| |
| [Download preset](https://cdn.discordapp.com/attachments/401099306167435269/875851587971592222/Persona_5_Nighttab_final.json) |
| [Reddit post](https://www.reddit.com/r/nighttab/comments/p3uo9m/first_post_persona_5_inspired_setup/) |

| Random clouds |
|-|
| [![Random clouds](https://preview.redd.it/iqpjg5umxxk71.png?width=960&crop=smart&auto=webp&s=093b0ce3ebbe503ec26c4f5f88502bc987f808b9)](https://gist.githubusercontent.com/metruzanca/07ae0b08d1703bc1b8818c45a35107cb/raw/eb2573eb2dd59843716450a6d0d83367092d7de3/nighttab%2520brkfst.json)| |
| [Download preset](https://gist.githubusercontent.com/metruzanca/07ae0b08d1703bc1b8818c45a35107cb/raw/eb2573eb2dd59843716450a6d0d83367092d7de3/nighttab%2520brkfst.json) |
| [Reddit post](https://www.reddit.com/r/nighttab/comments/pg0rsn/new_work_dashboard_image_is_random_clouds_sky/) |

| Purple Cityscape |
|-|
| [![Purple Cityscape](https://preview.redd.it/wwnm5az8gyk71.png?width=960&crop=smart&auto=webp&s=90870641e5a28252c08f28dd76cadbf92dcbcfa6)](https://gist.githubusercontent.com/zombieFox/200389b6ca7dc5aa35b1109fb2949287/raw/311be126f98a4fe6a1628c406bbedfdf04793831/gistfile1.txt)| |
| [Download preset](https://gist.githubusercontent.com/zombieFox/200389b6ca7dc5aa35b1109fb2949287/raw/311be126f98a4fe6a1628c406bbedfdf04793831/gistfile1.txt) |
| [Reddit post](https://www.reddit.com/r/nighttab/comments/pg325t/purple_cityscape_my_current_set_up/) |

**Comment by zombieFox** [comment]
Updated links 🔗 🚀

**Comment by Code2dpy** [comment]
Can we also add our presets here in case we do not have reddit?

**Reply by spa5k** [reply]
> Can we also add our presets here in case we do not have reddit?

Yeah, as long as you've a picture + settings link

**Reply by Code2dpy** [reply]
Oh OK, Thank you very much!

**Reply by zombieFox** [reply]
👍🏽 @Unbuttun 
Everyone is welcome to share.

**Reply by spa5k** [reply]
> 👍🏽 @Unbuttun 
> Everyone is welcome to share.

You can also pin this issue, so rest can see or open it I guess

**Reply by zombieFox** [reply]
Pinned 📍

**Comment by Code2dpy** [comment]
# Preset with random background
![2021-09-16](https://user-images.githubusercontent.com/84425771/133558368-658063e3-1f42-4ac8-8281-6191ed07d7e7.png)
[Download this preset (Note: this downloads as .txt so you will have to save as .json)](https://github.com/zombieFox/nightTab/files/7175243/nt.txt)

**Reply by zombieFox** [reply]
Awesome setup! 🎸

**Reply by Code2dpy** [reply]
Thank you so much

**Reply by spa5k** [reply]
Reminds me of old windows 8 and 8.1

**Comment by Gogeta999** [comment]
![image](https://user-images.githubusercontent.com/46393235/148397693-f1c0999d-e3a6-4b5f-834a-7d4248fcba40.png)

[Copy it and use import from clipboard method](https://gist.github.com/Gogeta999/2a8636c89b9cf41533ccc5525ab970f6)
Hope You enjoy this

**Reply by LordGrim9987** [reply]
@spa5k How to use this?

**Reply by spa5k** [reply]
> @spa5k How to use this?
> 

Install the extension, then open the extension settings - > data - > copy from clipboard option.

Extension name is - NightTab

**Reply by spa5k** [reply]
hey @Gogeta999 it seems like wherever you have uploaded it is saying access denied, you can upload it to gist.github.com and share the link for the json file from there.

**Reply by Gogeta999** [reply]
> hey @Gogeta999 it seems like wherever you have uploaded it is saying access denied, you can upload it to gist.github.com and share the link for the json file from there.

Sry let me fix it

**Reply by Gogeta999** [reply]
> @spa5k How to use this?

@LordGrim9987 Guy, i fix it, now you can copy json at gist.github and import like spak5k said.

**Comment by zombieFox** [comment]
A collection of [custom themes](https://github.com/hippostars/nightTab-Themes) by @hippostars 🦛 ⭐ 

[![937983092](https://user-images.githubusercontent.com/7115017/153752344-4633234d-e41d-48fe-90e7-8d0af059a17e.png)](https://github.com/hippostars/nightTab-Themes)
